x/gamm/pool-models/stableswap/util_test.go

```diff
@@ -0,0 +1,34 @@
+package stableswap
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/store/rootmulti"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/log"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
+
+	"github.com/osmosis-labs/osmosis/v7/x/gamm/types"
+)
+
+func createTestPool(t *testing.T, poolLiquidity sdk.Coins, swapFee, exitFee sdk.Dec) types.PoolI {
+	pool, err := NewStableswapPool(1, PoolParams{
+		SwapFee: swapFee,
+		ExitFee: exitFee,
+	}, poolLiquidity, "")
+
+	require.NoError(t, err)
+
+	return &pool
+}
+
+func createTestContext(t *testing.T) sdk.Context {
```

Comment:
I think this function is being shared by both pool packages. Can we instead move it to the common package and reuse?

Reply:
done in the latest commit